pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -379,6 +387,10 @@ func (b *BridgeLibvirtSpecGenerator) generate(domainIface api.Interface) error {
 	return nil
 }
 
+func (b *BridgeLibvirtSpecGenerator) decorateDomainIfaceSpec(domainIface *api.Interface) error {
+	return nil
```

Comment:
I don't understand. Is this PR reviewable ?

Reply:
Well as you can see in the title of the PR, it is a *draft*, so to answer your question- it is not ready to be reviewed.
I think that is what draft means, isn't it?

Just asked you opinion about the *concept* in the PR.

Regarding the method here, it's purpose is to decorate the `domainIface` with `phase2` information.
Since the PR is still WIP and the brdige binding part wasn't implemented yet, it returns nil.